src/scss/_breadcrumb.scss

```diff
@@ -76,3 +83,7 @@
 .jazz-breadcrumb--flush-left {
   margin-left: -10px;
 }
+
+.jazz-breadcrumb-spacer {
```

Comment:
It seems like this spacer would only be necessary when using an icon.  Is there any reason you cannot just add margin-right or padding-right to the icon element (.jazz-breadcrumb > ol > li > i) and eliminate the spacer div and class altogether?